### PR TITLE
Introduced the bd_noautoreload cvar

### PR DESCRIPTION
**Pull request by Zodomaniac**
so that one doesn't have to e.g. disable auto-reloading in non-tactical mode every map start.
Both disabling all reloading and auto-reloading take immediate effect.

### CODE REVIEW

**Comment by Zodomaniac** [comment]
I added the Delay and Restart lines to make the change take effect in-game, not just on map start.
Maybe you know a better way of doing that?

**Comment by Zodomaniac** [comment]
Where in DECORATE should I put the calls to the new scripts?

**Comment by Zodomaniac** [comment]
Managed to both apply the settings immediately and read them on map start.

**Comment by BLOODWOLF333** [comment]
The way MENUDEF is currently allows you to give/take turboreload and noautoreload immediately in-game on single player.  That option is using console commands.
`Command "<label>", "<command>"
Calls a console command.`
I created that bd_disablereloading cvar because give and take console commands are cheats online so, you cant change change them with an immediate effect online.  There is no way.

File: KEYCONF.txt [hunk 1]
```diff
@@ -10,4 +10,10 @@ setslot 6 ClassicPlasmaRifle Plasma_Gun RailGun
 setslot 7 NukeLauncher BFG10k VanillaBFG9000 BIG_FUCKING_GUN
 setslot 8 Unmaker HellishMissileLauncher
 setslot 9 FlameCannon Flamethrower2 
-setslot 0 HandGrenades
\ No newline at end of file
+setslot 0 HandGrenades
```
Comment (BLOODWOLF333) [inline KEYCONF.txt:13]: We dont need these edits

File: MENUDEF.TXT [hunk 2]
```diff
@@ -186,16 +186,19 @@ OptionMenu "BrutalDoomOptionsGame"
...
    StaticText " "
    Option "Rocket Launcher Backblast", "bd_rocketbackblast", "YesOrNo"
+   StaticText " "
   StaticText "When enabled, you will harm yourself if using the"
   StaticText "Rocket Launcher with insufficient clearance behind you."
   StaticText " "
-   Command "Enable Reloading",			"take turboreload"
-   Command "Skip Reloading",			"give turboreload"
-   StaticText "If using ''Skip Reloading'', reload animations"
+   Command "Enable Reloading",			"enablereloading"
```
Comment (BLOODWOLF333) [inline MENUDEF.TXT:193]: https://zdoom.org/wiki/MENUDEF
`Command "<label>", "<command>"
Calls a console command.`
"enablereloading" is NOT a console command take <turboreload> is a console command to take the turboreload token.  This is wrong.

Comment (Zodomaniac) [reply MENUDEF.TXT:193]: In KEYCONF, I added the aliases like:
alias disableautoreload "bd_disableautoreload 1; give noautoreload"
as on `Command "Enable Reloading"` I need to execute 2 commands: `bd_disableautoreload 1` and `give noautoreload`. Will it work without the alias?

Comment (BLOODWOLF333) [reply MENUDEF.TXT:193]: yeah we dont need those alias

**Comment by Zodomaniac** [comment]
Done.

File: MENUDEF.TXT [hunk 3]
```diff
@@ -190,14 +190,14 @@ OptionMenu "BrutalDoomOptionsGame"
   StaticText "When enabled, you will harm yourself if using the"
   StaticText "Rocket Launcher with insufficient clearance behind you."
   StaticText " "
-   Command "Enable Reloading",			"enablereloading"
-   Command "Skip Reloading",			"disablereloading"
+   Command "Enable Reloading",			"bd_disablereloading 0; take turboreload"
```
Comment (BLOODWOLF333) [inline MENUDEF.TXT:193]: get rid of the bd_disablereloading 0; lines you simply need just "take turboreload" this is the same thing as going into the console (~) and typing "take turboreload".  I don't think your getting this?

Comment (BLOODWOLF333) [reply MENUDEF.TXT:193]: image
The parameters is where bd_disablereloading 1 would go when hosting a game online

**Comment by Zodomaniac** [comment]
Ok, done.

**Pull request merged**